### PR TITLE
pulled submission directories out of zulip-gci

### PR DESCRIPTION
**Pull request by Lyla-Fischer**

### CODE REVIEW

**Comment by Lyla-Fischer** [comment]
I kept the bot-avatars directory, even though we aren't committed to that set of tasks yet, because I didn't want to search through git histories to get the samples in case we resurrected those tasks.

**Comment by Lyla-Fischer** [comment]
Also, all of the README's are still using last year's task descriptions, and they will need updated once that is relevant, but I figured this was a good start.

**Comment by hackerkid** [comment]
@Lyla-Fischer Kind of curious why there is a full task description for translations directory while all other READMEs have something like `+../tasks/2016/bot-avatars.md`. We don't have plans to include task descriptions in this repo right?

**Comment by Lyla-Fischer** [comment]
I honestly didn't realize that they were symlinks. (I was just copy-pasting). No, we don't intend to include task descriptions in this repo. I've updated this to use hyperlinks instead of symlinks.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,118 @@
...
+environment](#setting-up-the-zulip-development-environment).
+
+Once you're done, we recommend browsing through Zulip's [extensive
+documentation](http://zulip.readthedocs.io/en/latest/readme-symlink.html) about
+its code and community. If you're planning on submitting a lot of code, we
+particularly recommend the guides to
+[git](https://zulip.readthedocs.io/en/latest/git-guide.html), [version
+control](https://zulip.readthedocs.io/en/latest/version-control.html), [coding
+style](https://zulip.readthedocs.io/en/latest/code-style.html), and
+[testing](https://zulip.readthedocs.io/en/latest/testing.html).
```
Comment (timabbott) [inline README.md]: I think we actually want the README.md content to stay in the zulip-gci repository; this should just have a brief version that links over to that.  My reasoning is that our strategy is to have all the actual documentation content in zulip-gci.git, with this repository just containing the year's student submissions.

So I think you should submit this README.md changeset as an update to the existing zulip-gci repo?

**Comment by timabbott** [comment]
Nice, I posted one comment (on how the README.md should probably go into zulip-gci.git); other than that, this looks great!

**Pull request merged**